dali/util/pybind.h

```diff
@@ -231,5 +231,36 @@ static DLMTensorPtr DLMTensorPtrFromCapsule(py::capsule &capsule) {
   return {DLMTensorRawPtrFromCapsule(capsule), DLMTensorPtrDeleter};
 }
 
+
+/**
```

Comment:
I guess if you export the PipelineOutputDesc this won't be necessary. I am not sure if it's worth to add the code with more comments rather than just using get<i> by hand either way.

Reply:
I'll have a go with `@dataclass`, we'll see if this will stay or not